### PR TITLE
Changing Freighter Weapons Slot for adding a Minin Device Slot

### PR DESCRIPTION
**Pull request by XxSARGExX**
All Freighter Medium Slots are changed to Heavy Slots

Exception: Mule

Mining Device Added on following Slots:

Rhino: HpWeapon01
Clydestale: HpWeapon01
Humpback: HpTurret05
Drone: HpTurret06
Mule: HpTurret01
Dromedary: HpWeapon01

Armored Transport: HpTurret07

### CODE REVIEW

File: DATA/SHIPS/shiparch.ini [hunk 1]
```diff
@@ -2269,13 +2270,14 @@ ids_info = 461646
...
 ; 
 ; \bWeapon Mounts\B
-; Guns/Launchers: 2 M, 1 H
+; Guns/Launchers: 2 H
 ; Turrets: 6
 ; Droppers: 1 Mine, 1 CM
 ; 
 ; \bExternal Mounts\B
-; Shields: -
+; Shields: 1X
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:2278]: 1 X with a space between

File: DATA/SHIPS/shiparch.ini [hunk 2]
```diff
@@ -2292,7 +2294,7 @@ ids_info3 = 461647
 ; $shieldBatteries/$nanobots
 ; 
 ; 
-; 2 M, 1 H
+; 2 H
 ; 6
 ; 1 Mine, 1 CM
 ;
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:2300]: 1 X Shield and Mining Device are not listed in the lines below here

File: DATA/SHIPS/shiparch.ini [hunk 3]
```diff
@@ -3886,7 +3889,7 @@ ids_info3 = 461671
 ; $shieldBatteries/$nanobots
 ; 
 ; 
-; 2 M, 1 H
+; 2 H
 ; 5
 ; 1 Mine, 1 CM
 ;
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:3895]: Mining Device missing in all the "reduced" statistics

**Review by erikroe — CHANGES_REQUESTED** [review]

**Review by erikroe — CHANGES_REQUESTED** [review]
ids_info2 for all the Freighters with Mining Equipment  need the version where Mining Equipment is listed. See the top of the shiparch.ini file.

Also, the Mining Ship and CSV both do not have Mining Equipment listed. They have them instead as 1 X gun in their stats. Needs to be adjusted.

File: DATA/SHIPS/shiparch.ini [hunk 4]
```diff
@@ -1553,7 +1557,8 @@ ids_info3 = 461639
 ; 
 ; -
 ; -
-; 
+; 1
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:1560]: Mining Device added here, but you forgot to remove the 1 X above

File: DATA/SHIPS/shiparch.ini [hunk 5]
```diff
@@ -14,6 +14,7 @@
 ; \bExternal Mounts\B
 ; Shields:
 ; Thrusters:
+; Mining Device:
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:17]: This is just an incomplete solution. Now you added this to every ship's statistics. So if you go for this, _every_ ship must now have a stats entry for Mining Devices.
Or make a copy of this stats entry for just those ships that have actual mining devices.

**Review by erikroe — CHANGES_REQUESTED** [review]

File: DATA/SHIPS/shiparch.ini [hunk 6]
```diff
@@ -3886,14 +3952,14 @@ ids_info3 = 461671
...
 ; 
-; 2 M, 1 H
+; 2 H, 1 Mining
 ; 5
 ; 1 Mine, 1 CM
 ; 
 ; 
 ; 1 X
 ; 1
-;
```
Comment (Caenen) [inline DATA/SHIPS/shiparch.ini:3896]: Space missing

File: DATA/SHIPS/shiparch.ini [hunk 7]
```diff
@@ -3886,14 +3952,14 @@ ids_info3 = 461671
...
-; 2 M, 1 H
+; 2 H, 1 Mining
 ; 5
 ; 1 Mine, 1 CM
 ; 
 ; 
 ; 1 X
 ; 1
-; 
+;
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:3962]: There is a space missing here. It must be `; `

**Review by erikroe — CHANGES_REQUESTED** [review]

**Review by erikroe — APPROVED** [review]

**Pull request merged**